packages/cloud_firestore/cloud_firestore/lib/src/load_bundle_task.dart

```diff
@@ -0,0 +1,13 @@
+part of cloud_firestore;
+
+class LoadBundleTask {
+  LoadBundleTask._(this._delegate) {
+    LoadBundleTaskPlatform.verifyExtends(_delegate);
+  }
+
+  final LoadBundleTaskPlatform _delegate;
+
+  Stream<LoadBundleTaskSnapshot> stream() {
```

Comment:
Would it make sense to have:

```dart
late final Stream<LoadBundleTaskSnapshot> stream =  _delegate.stream.map((event) => LoadBundleTaskSnapshot._(event));
```

?

Reply:
I've left this. If you feel that it is worth changing at this stage, let me know and I'll update.

Reply:
I think that's worth doing now. As later it would be a breaking change, and it's fairly straightforward to do.